src/site/controllers/WebController.php

```diff
@@ -34,14 +32,11 @@ abstract class WebController {
 
   public function __construct(
     ImmMap<string, string> $parameters,
-    private ServerRequestInterface $request,
+    private Message\IServerRequest $request,
   ) {
     $combined_params = new Map($parameters);
     foreach ($request->getQueryParams() as $key => $value) {
-      if (is_array($value)) {
-        continue;
-      }
-      $combined_params[(string)$key] = (string)$value;
+      $combined_params[$key] = $value;
```

Comment:
Would this now include the normalized array strings in the arguments?
Previously, they were ignored.

Reply:
yes. but that's not an issue in this case.